src/main/java/com/assignment/application/kafka/KafkaConfig.java

```diff
@@ -33,69 +35,82 @@
 @Configuration
 public class KafkaConfig {
 
-    Logger logger = LoggerFactory.getLogger(KafkaConfig.class);
+    private static final Logger LOG = LogManager.getLogger(KafkaConfig.class);
 
     @Autowired
     private StringConstant stringConstant;
 
-
     @Bean
-    public ProducerFactory<String,EmployeeInfoUpdate> producerFactoryEmpUpdate(){
-        Map<String,Object> map = new HashMap<>();
+    public ProducerFactory<String, EmployeeInfoUpdate> producerFactoryEmpUpdate() {
+        Map<String, Object> map = new HashMap<>();
         map.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, StringConstant.HOSTNAME);
         map.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
         map.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonSerializer.class);
-        return new DefaultKafkaProducerFactory<String,EmployeeInfoUpdate>(map);
+        return new DefaultKafkaProducerFactory<String, EmployeeInfoUpdate>(map);
     }
 
     @Bean
-    public KafkaTemplate<String, EmployeeInfoUpdate> kafkaTemplateEmpUpdate(){
+    public KafkaTemplate<String, EmployeeInfoUpdate> kafkaTemplateEmpUpdate() {
         return new KafkaTemplate(producerFactoryEmpUpdate());
     }
 
     @Bean
-    public ProducerFactory<String, Employee> producerFactoryEmp(){
-        Map<String,Object> map = new HashMap<>();
+    public ProducerFactory<String, Employee> producerFactoryEmp() {
+        Map<String, Object> map = new HashMap<>();
         map.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, StringConstant.HOSTNAME);
         map.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
         map.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonSerializer.class);
-        return new DefaultKafkaProducerFactory<String,Employee>(map);
+        return new DefaultKafkaProducerFactory<String, Employee>(map);
     }
 
     @Bean
-    public KafkaTemplate<String, Employee> kafkaTemplateEmp(){
+    public KafkaTemplate<String, Employee> kafkaTemplateEmp() {
         return new KafkaTemplate(producerFactoryEmp());
     }
 
     @Bean
-    public ProducerFactory<String, SalaryUpdate> producerFactoryEmpSalary(){
-        Map<String,Object> map = new HashMap<>();
+    public ProducerFactory<String, SalaryUpdate> producerFactoryEmpSalary() {
```

Comment:
Because I have different model for different topic. Employee model class for employee information topic and salary update class for salary update topic.

Reply:
Declare those Configuration in a function and call the function as required